exchange/exchange.go

```diff
@@ -447,11 +459,13 @@ func applyCategoryMapping(ctx context.Context, requestExt openrtb_ext.ExtRequest
 					if dupe.bidderName == bidderName {
 						// An older bid from the current bidder
 						bidsToRemove = append(bidsToRemove, dupe.bidIndex)
+						rejections = updateRejections(rejections, dupe.bidID, "Bid was deduplicated")
```

Comment:
Consider using `bidID` for consistency with the other calls..

Reply:
This is correct as is in this case, we are removing dupe.bidID instead of bidID (these are two different bids).

Reply:
>  The removal of bids uses the index instead of the ID

That's the part I was missing.